lib/pos_blockchain.go

```diff
@@ -1722,6 +1722,11 @@ func (bc *Blockchain) commitBlockPoS(blockHash *BlockHash) error {
 	return nil
 }
 
+// GetCommittedTipView builds a UtxoView to the committed tip.
+func (bc *Blockchain) GetCommittedTipView() (*UtxoView, error) {
+	return NewUtxoView(bc.db, bc.params, nil, bc.snapshot, nil)
```

Comment:
we should probably supply bc.postgres here in the event that this function gets used in places that require pg.